docs/source/Usage.md

```diff
@@ -147,6 +147,7 @@ The following is the list of the available analyzers you can run out-of-the-box:
 Some Analyzers require details other than just IP, URL, Domain etc... We classified them as Generic Analyzers. Since the type of field is not known, there is a format for strings to be followed.
 * `EmailRep`: search an email address on emailrep.io
 * `WiGLE`: Maps and database of 802.11 wireless networks, with statistics, submitted by wardrivers, netstumblers, and net huggers.
+* `CRXcavator`: CRXcavator automatically scans the entire Chrome Web Store every 3 hours and produces a quantified risk score for each Chrome Extension based on several factors.
```

Comment:
this is the description of the service, not the description of what you can do with this analyzer